aegisthus-distcp/src/main/java/Distcp.java

```diff
@@ -215,32 +219,51 @@ protected void setupTempDirs() {
 	}
 
 	protected void setReducers(Job job, int fileCount) {
-		int reducers = job.getConfiguration().getInt("mapred.reduce.tasks", 1);
+		int reducers = job.getConfiguration().getInt("mapred.job.reduces", 1);
```

Comment:
The former is deprecated, and for some reason it was set to 23 when I ran it on the long-running cluster, while the non-deprecated one is set to 1, which is consistent with the defaults in hadoop's documentation.  If the deprecated and non-deprecated configuration fields can simultaneously store distinct values, we need to be better about avoiding deprecated fields.